poh/src/poh_recorder.rs

```diff
@@ -570,12 +567,14 @@ impl PohRecorder {
                 hash: poh_entry.hash,
                 transactions: vec![],
             };
-            self.tick_overhead_us += timing::duration_as_us(&now.elapsed());
 
-            let now = Instant::now();
-            self.tick_cache.push((entry, self.tick_height));
-            let _ = self.flush_cache(true);
-            self.flush_cache_tick_us += timing::duration_as_us(&now.elapsed());
+            {
+                let now = Instant::now();
+                self.tick_cache.push((entry, self.tick_height));
+                let _ = self.flush_cache(true);
+                self.flush_cache_tick_us += timing::duration_as_us(&now.elapsed());
+            }
+
```

Comment:
maybe could use `Measure::this` for these scopes: https://github.com/solana-labs/solana/pull/16953/files